website/docs/developer/research/research-bamboo-integration-tests.md

```diff
@@ -0,0 +1,129 @@
+---
+id: research-bamboo-integration-tests
+title: Research Notes on running integration tests in bamboo CI/CD
+description: Research Notes on deploying Cumulus and ORCA in bamboo CI/CD and running integration tests.
+---
+
+## Deploying terraform modules via bamboo
+AWS resources can be deployed via Bamboo pipeline by first adding the following to a script and running that in a Bamboo task.
+
+```bash
+#configure aws 
+aws configure set aws_access_key_id $bamboo_AWS_ACCESS_KEY_ID
```

Comment:
If you have the values for AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and AWS_DEFAULT_REGION you should not need to run the aws configure commands just do an export.

```
export AWS_ACCESS_KEY_ID=$bamboo_AWS_ACCESS_KEY_ID
export AWS_SECRET_ACCESS_KEY=$bamboo_AWS_SECRET_ACCESS_KEY
export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
```

See https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

Reply:
yes tested and looks good. 
updated bin/deployment-cumulus-orca.sh and bin/deployment-rds.sh

Reply:
You may also want to update your documentation as well in this document to show them as exports.

Reply:
oops forgot to do that. Updated